src/modules/models/Garden.tsx

```diff
@@ -0,0 +1,21 @@
+import { RoundedBox } from '@react-three/drei';
+
+import { Seasons, useSeason } from '@/context/SeasonContext';
+
+const COLORS_BY_SEASON = {
+  [Seasons.Winter]: '#D0D4D8',
+  [Seasons.Spring]: '#76B041',
+  [Seasons.Summer]: '#52b041',
+  [Seasons.Fall]: '#94b041',
+};
+
+export const Garden = (): JSX.Element => {
+  const SIZE = 16;
```

Comment:
Are these in any form of dimensions ? meters, cm ?

Reply:
We have the responsibility to choice a dimension. I precise in the const that its in meter.